.github/workflows/snyk-python-scan.yml

```diff
@@ -0,0 +1,55 @@
+
+name: 🚨 Snyk Python Scan
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 3'
+
+concurrency:
+  # SHA is added to the end if on `main` to let all main workflows run
+  group: ${{ github.ref }}-${{ github.workflow }}-${{ github.event_name }}-${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/release/') || startsWith(github.ref, 'refs/heads/long_lived/')) && github.sha || '' }}
+  cancel-in-progress: false
```

Comment:
Why not cancel in progress runs on the same branch like we do everywhere else in this repo?